mmv1/third_party/terraform/tests/resource_storage_bucket_object_test.go

```diff
@@ -359,13 +393,21 @@ func TestAccStorageObject_holds(t *testing.T) {
 	})
 }
 
-func testAccCheckGoogleStorageObject(t *testing.T, bucket, object, md5 string) resource.TestCheckFunc {
+func testAccCheckGoogleStorageObject(t *testing.T, bucket, object, md5 string, customerEncryptionKey string) resource.TestCheckFunc {
```

Comment:
lets add a wrapper instead of passing `""` to the majority of the function calls

```suggestion
func testAccCheckGoogleStorageObject(t *testing.T, bucket, object, md5 string) resource.TestCheckFunc {
	return testAccCheckGoogleStorageObjectWithEncryption(t, bucket, object, md5, "")
}

func testAccCheckGoogleStorageObjectWithEncryption(t *testing.T, bucket, object, md5 string, customerEncryptionKey string) resource.TestCheckFunc {
```

Reply:
Good idea, done